### DISCUSSION TITLE
[Deathdusk modpack] Vanilla Recipe Book not working

### DISCUSSION THREAD

**Discussion by jony-test** [shedaniel/RoughlyEnoughItems#888] [Q & A] [answered]
I suppose this option is for allowing or removing it. Right now it's staying removed regardless of settings

[enabled](https://i.imgur.com/DarhCbC.jpeg)
[disabled](https://i.imgur.com/MokZqUf.jpeg)

**Comment by shedaniel** [comment, marked as answer]
Your recipe book button doesn’t look like the vanilla one, I am guessing that you have another mod that replaces the vanilla book, so that’s why the REI option would do nothing.
